clients/src/main/resources/common/message/UpdateRaftVoterResponse.json

```diff
@@ -23,6 +23,16 @@
     { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
     { "name": "ErrorCode", "type": "int16", "versions": "0+",
-      "about": "The error code, or 0 if there was no error" }
+      "about": "The error code, or 0 if there was no error" },
+    { "name": "CurrentLeader", "type": "CurrentLeader", "versions": "0+",
```

Comment:
It seems like `RequestResponseTest.createUpdateRaftVoterResponse` has not been updated to set these new fields.

Reply:
Fixed.